### PR TITLE
reading nullable Guids

### PR DESCRIPTION
**Pull request by andriy-viyatyk**

### CODE REVIEW

File: tiberius/src/tokens.rs [hunk 1]
```diff
@@ -457,6 +457,7 @@ impl<I: Io> ParseToken<I> for TokenNbcRow {
             // calculate size of the null-bitmap and read it
             let bitmap_size = (col_meta.columns.len() + 8 - 1) / 8;
             trans.inner.row_bitmap = trans.inner.read_bytes(bitmap_size);
+            trans.inner.commit_rd_buffer();
```
Comment (steffengy) [inline tiberius/src/tokens.rs]: could you add any test (or provide a sample) to reproduce the crash.
this fix feels wrong, it probably skips data.
The actual problem is likely some token implementation that is not reading all data.

Comment (andriy-viyatyk) [reply tiberius/src/tokens.rs]: [tiberius_test.zip](https://github.com/steffengy/tiberius/files/2255004/tiberius_test.zip)

Comment (andriy-viyatyk) [reply tiberius/src/tokens.rs]: P.P.S. On provided test data You should first reproduce "nullable Guids problem" (my first commit), fix it, and then should reproduce the crush.

Comment (steffengy) [reply tiberius/src/tokens.rs]: Yeah works, the fix addresses the actual issue (the bitmap being read as column data when multiple reads are necessary).
So all good, thanks for the repro.

**Pull request merged**